plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/e4/DBeaverStackRenderer.java

```diff
@@ -271,6 +275,22 @@ public void widgetSelected(SelectionEvent e) {
                         DBWorkbench.getPlatformUI().copyTextToClipboard(DBUtils.getObjectFullName(object, DBPEvaluationContext.UI), false);
                     }
                 });
+                if (workbenchPart instanceof EntityEditor) {
+                    final MenuItem addBookmarkItem = new MenuItem(menu, SWT.NONE);
+                    addBookmarkItem.setText(NLS.bind(ADD_BOOKMARK_COMMAND_NAME, label));
```

Comment:
1. Why do we need ADD_BOOKMARK_COMMAND_NAME as a static variable?
2. Add just command ID to CoreCommands.
3. What NLS.bind does here? 
4. Let's also add icon and shortcut info in menu item.